dashboard/test/models/school_info_test.rb

```diff
@@ -435,3 +435,86 @@ class SchoolInfoTest < ActiveSupport::TestCase
     assert_equal 'Custom School', school_info_custom_school.effective_school_name
   end
 end
+
+# Tests following spec:
+# We should pop up this interstitial again every 7 days unless one of the following is true:
```

Comment:
This comment is a little confusing since these are `SchoolInfo` tests, not interstitial tests.